### ISSUE TITLE
Can't invalidate Vipps Jwt Token. Please try again.

### ISSUE THREAD

**Issue by DigiBlooD** [vippsas/vipps-magento#9]
Hi

Have talked to Vipps support but I think its easier to just take this problem here.

When everything is installed and configured with keys etc. and "Close" and "save Config" is pressed I get the following error message "Can't invalidate Vipps Jwt Token. Please try again.".  Same error is given on a clean Magento 2.2.5 instance.

The DB is: jens_petrus
DB Prefixen is:  jens_

Gives following error in vipps_debug.log when I tried last week before talking to Support.
```
[2018-08-19 11:48:36] vipps.CRITICAL: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jens_petrus.vipps_payment_jwt' doesn't exist, query was: SELECT `vipps_payment_jwt`.* FROM `vipps_payment_jwt` WHERE (scope_id = 1) ORDER BY `token_id` DESC
LIMIT 1 [] []
[2018-08-19 11:49:57] vipps.CRITICAL: PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jens_petrus.vipps_payment_jwt' doesn't exist in /home/petrus/public_html/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php:228
```

Tried regenerating the keys today as recommended by support, did not help. Error is the same and here is the info from the debug log.
```
[2018-08-24 12:21:10] vipps.CRITICAL: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jens_petrus.vipps_payment_jwt' doesn't exist, query was: DELETE FROM `vipps_payment_jwt` WHERE (scope_id = 1) [] []
```

I have checked the DB and jens_vipps_payment_jwt is there, but it seems Vipps module is trying to fetch it without the prefix vipps_payment_jwt.

Any recommendations?

```
**Preconditions**
Magento v2.2.5
PHP v7.1.19 
bcmath
dba
dom
enchant
fileinfo
gd
imagick
imap
intl
json
ldap
mbstring
mcrypt
mysqli
mysqlnd
oauth
odbc
opcache
pdo
pdo_odbc
pdo_pgsql
pdo_sqlite
pgsql
phar
posix
pspell
snmp
soap
sockets
sysvmsg
sysvsem
sysvshm
timezonedb
wddx
xmlreader
xmlrpc
xmlwriter
xsl
zip
MySQL v10.1.31-MariaDB-cll-lve
Apache v2.4.33
Perl v5.16.3
Kernel v3.10.0-714.10.2.lve1.5.12.el7.x86_64

```

```
**Composer Installation Log:**

composer require vipps/module-payment
Using version ^1.0 for vipps/module-payment
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing vipps/module-payment (1.0.2): Loading from cache
Package fabpot/php-cs-fixer is abandoned, you should avoid using it. Use friendsofphp/php-cs-fixer instead.
Writing lock file
Generating autoload files

./bin/magento module:enable Vipps_Payment
The following modules have been enabled:
- Vipps_Payment

To make sure that the enabled modules are properly registered, run 'setup:upgrade'.
Cache cleared successfully.
Generated classes cleared successfully. Please run the 'setup:di:compile' command to generate classes.
Info: Some modules might require static view files to be cleared. To do this, run 'module:enable' with the --clear-static-content option to clear them.


./bin/magento setup:upgrade
Cache cleared successfully
File system cleanup:
/home/petrus/public_html/generated/code/Composer
/home/petrus/public_html/generated/code/Magento
/home/petrus/public_html/generated/code/Symfony
/home/petrus/public_html/pub/static/_cache
/home/petrus/public_html/pub/static/adminhtml
/home/petrus/public_html/pub/static/deployed_version.txt
/home/petrus/public_html/pub/static/frontend
/home/petrus/public_html/var/view_preprocessed/pub
Updating modules:
Schema creation/updates:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Backend':
Module 'Magento_Theme':
Module 'Infortis_Infortis':
Module 'Infortis_Brands':
Module 'Infortis_Cgen':
Module 'Infortis_Dataporter':
Module 'Infortis_Base':
Module 'Magento_Customer':
Module 'Infortis_UltraSlideshow':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Module 'Magento_Cms':
Module 'Magento_Catalog':
Module 'Magento_Quote':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Authorization':
Module 'Magento_Rule':
Module 'Magento_CatalogRule':
Module 'Magento_Backup':
Module 'Magento_SalesSequence':
Module 'Magento_Bundle':
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Payment':
Module 'Magento_CatalogInventory':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Module 'Magento_Widget':
```

**Comment by DrKoolaid** [comment]
I think the plugin dont support database prefix. 

For now can maybe just add ur prefix in the code

**Comment by DrKoolaid** [comment]
I remember now.., I just added the vipps table without the prefix.

**Comment by voleye** [comment]
@DigiBlooD thank you for your request, yes, this is a bug, we will fix it and get back to you soon

**Comment by cloveras** [comment]
FYI, @DigiBlooD and @DrKoolaid: I have closed the Vipps case, as this is now being handled here.

**Comment by ed007m** [comment]
@DigiBlooD , @DrKoolaid 
Thank you for reporting. This issue was fixed in 1.0.6 release.
